Issue & PR templates

kirstyburgoine opened the pull request:
## Description

**Templates for issue creation.** 
Although the motivation for these is for outside contributors to follow a specific format, templates have been created for all of the task labels so that we use them internally as well. Only the task `Spike` doesn't have a template because we pretty much have all of the Spike's outlined now, and it's not expected that third parties would add any more of these.

These are based on the Github defaults with only a few small changes depending on task type.

**Pull request template**
Default pull request template for all types of work.

kirstyburgoine committed:
Bug & Feature report issue templates

Templates have been created for all labels for `types` except for `Spike` as we pretty much already have those now and it's not expected that third parties would any more of these.

These are based on the Github defaults with only a few small changes.

svandragt commented:
What is the difference between  a bug-report and a type-bug report in your PR? I don't understand why it's duplicated.

kirstyburgoine commented:
@svandragt Ah, nothing. Github created this PR automatically and I wasn't quick enough to change it to draft. I still have more to do here

kirstyburgoine committed:
Delete duplicate templates with wrong names

The issue templates were created in Github under settings and branch / PR was created automatically when committing changes. These files were an autosave before the templates were renamed to match labels

kirstyburgoine committed:
Add acceptance criteria section to issue templates

kirstyburgoine committed:
Basic pull request template

kirstyburgoine committed:
Change wording on PR template to match others

joemcgill reviewed (approved):
I think these are a great starting point. Let's give it a go and make changes based on real usage.
